### ISSUE TITLE
When I add an ng-include, everything gets repeated (¿?)

### ISSUE THREAD

**Issue by ladaltamirano** [ngbp/ngbp#288]
When I add

``` html
<div ng-include="'partials/navbar.tpl.html'"></div>
```

in line 91, index.html , everything gets repeated! Try it and you'll see!... Why this happens?
How can I use or add partials views? Thanks..
(Yeah, that folder and its file exist)

**Comment by joshdmiller** [comment]
`*.html` aren't picked up by the build be default. Use `navbar.tpl.html` or add the pattern to your `build.conf.js` file.

**Comment by ladaltamirano** [comment]
Ok, done, but still repeats a lot of sections quite a lot of times!... Has any of you already noticed it? :)

**Comment by joshdmiller** [comment]
No, I use partials without issue. Can you push a minimal example up to github so we can check it out?

**Comment by gregra81** [comment]
Can you advise on the changes we need to make in build.config.js/Grunfile.js in order so support a partial folder inside build and compile (for example, this is handy for ng-include and templateUrl in a directive since they both fetch the template with ajax).
Or am I missing something and this can be achieved in some other way?

Thanks in advance

**Comment by joshdmiller** [comment]
@gregra81 As long as you use the correct naming convention (see my earlier comment), nothing need be done. ngbp uses `html2js` during the build, so all templates (directives, partials, routes, etc.) are put into a JS file that is added as a `script` tag in `index.html`; with ngbp, there are no AJAX requests for templates.

If you _want_ to fetch them asynchronously, you'll need to get the templates there, such as with a new `copy` task that takes whatever pattern you want, e.g. `src/app/**/*.partial.html`, and copies the files to `build_dir/assets/templates` or some such location.

**Comment by gregra81** [comment]
OK, cool. The key is to work with "assets" as the home folder for "partials" (since I haven't found a way to alter build.config.js/Grunfile.js to support working with  a "partials" folder on the root of "src").

Thanks!

**Comment by ladaltamirano** [comment]
Yeah, it worked great locating the partials at assets/ . Thanks Josh for enlightening us about html2js. Now I have this <%= version %> of the navbar that, being called in an include, it's not conveniently rendered (ngbp v<%= version %> ...) ..., where does this variable come from and how can I reference it from angular? Regards.
